components/raftstore/src/store/fsm/apply.rs

```diff
@@ -3127,6 +3127,10 @@ where
                     if channel_timer.is_none() {
                         channel_timer = Some(start);
                     }
+                    if let Some(timer) = channel_timer {
+                        let elapsed = duration_to_sec(timer.elapsed());
+                        APPLY_TASK_WAIT_TIME_HISTOGRAM.observe(elapsed);
```

Comment:
It will be observed several times. I think it should either be moved to L3128 or just remove `channel_timer`.

Reply:
Addressed